stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh

```diff
@@ -61,9 +61,12 @@ if [ -n "$1" ]; then
       ((i+=1))
   done
   # Mount BSP folder and pass it to the build script, if requested.
-  if [ -n "$BSP" ]; then
+  if [[ -n "$BSP" && -d "$BSP" ]]; then
     DOCKER_OPTS+="-v $BSP:/bsp-directory "
     CMD_OPTS+="--bsp-path /bsp-directory "
+  elif [[ $BSP =~ ^.*.tgz$ ]]; then
```

Comment:
Could this be `[[ -n "$BSP" && -f "$BSP" ]]` ?

> [ -f FILE ] True if FILE exists and is a regular file.

Reply:
Yes, this should work